_data/simple-icons.json

```
@@ -4228,6 +4228,11 @@
 		"hex": "3793EF",
 		"source": "https://deepnote.com"
 	},
+	{
+		"title": "Deepseek",
+		"hex": "4D6BFE",
+		"source": "https://chat.deepseek.com/favicon.svg"
```

There is a better source for the icon. Can you process the icon based on this one?

```suggestion
		"source": "https://github.com/deepseek-ai/DeepSeek-V2/blob/ec98ee3cbffc32104cd55dba8af884b3d772602a/figures/logo.svg"
```